Fix issue with the isModule property being ignored.

apokralipsa opened the pull request:
Fixes #123 

There might be more similar cases introduced when the `strict` mode was being enabled in `tsconfig.json`. 
I am not aware of them as I am mostly configuring elements using `ElementConfig`s.

tomastrajan commented:
hey there @apokralipsa !

Thank you very much for your effort, I have missed it before the A14 update so now there is a conflict, would be really interested in adding this.


Could you please update the PR ?

Thank you!

apokralipsa commented:
Hey @tomastrajan 
Rebased and fixed the conflicts.
Locally I have some issue with running `npm run test` now. It says I am missing some files matching `/projects/**/*.md`. 
Do I need to update some markdown manually?
